src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -46,6 +46,7 @@
         "symfony/form": "~2.8|~3.0",
         "symfony/expression-language": "~2.8|~3.0",
         "symfony/process": "~2.8|~3.0",
+        "symfony/serializer": "~2.8|^3.1",
```

Comment:
I have left this addition for dev requirement. It would appear that is was completely missing before.